tests/models/whisper/test_tokenization_whisper.py

```diff
@@ -204,11 +217,21 @@ def test_skip_special_tokens_skips_prompt_ids(self):
         # fmt: on
         expected_with_special_tokens = "<|startofprev|> Mr. Quilter<|startoftranscript|><|en|><|transcribe|><|notimestamps|> On the general principles of art, Mr. Quilter writes with equal lucidity.<|endoftext|>"
         expected_without_special_tokens = " On the general principles of art, Mr. Quilter writes with equal lucidity."
-        self.assertEqual(tokenizer.decode(encoded_input, skip_special_tokens=False), expected_with_special_tokens)
```

Comment:
Can you remove all these changes which shouldn't be applied (our line length is 120 and this is a formatting change unrelated to the PR)

Reply:
@amyeroberts All unrelated changes have been reverted. Now is it the proper time for merging the PR?